modules/vocabularies/src/main/frontend/src/vocabularyTable.jsx

```diff
@@ -17,56 +17,32 @@
 //  under the License.
 //
 
-import React from "react";
+import React, { useState } from "react";
 
 import {
   Grid,
-  makeStyles,
-  Paper,
-  Table,
-  TableBody,
-  TableCell,
-  TableHead,
-  TableRow,
+  IconButton,
   Typography,
   withStyles
 } from "@material-ui/core";
 
+import { useTheme } from '@material-ui/core/styles';
+import MaterialTable from "material-table";
+import VocabularyActions from "./vocabularyActions"
 import Search from "./search";
-import VocabularyEntry from "./vocabularyEntry";
 
-const Config = require("./config.json");
 const Phase = require("./phaseCodes.json");
 
-const useStyles = makeStyles(theme => ({
-  root: {
-    overflowX: "auto"
-  },
-  table: {
-    tableLayout: "fixed"
-  }
-}));
-
-// Creating customized Table components for a nicer look
-const HeaderTableCell = withStyles(theme => ({
-  head: {
-    backgroundColor: theme.palette.common.black,
-    color: theme.palette.common.white
-  }
-}))(TableCell);
-
 export default function VocabularyTable(props) {
-  const [filterTable, setFilterTable] = React.useState(false);
-  const [acronymList, setAcronymList] = React.useState([]);
-
-  const vocabList = props.vocabList;
-  const classes = useStyles();
-  const headingTypography = Config["tableHeadingTypography"];
-  const columnWidths = Config["tableColumnWidths"];
+  const { vocabList, type  } = props;
+  const [filterTable, setFilterTable] = useState(false);
+  const [acronymList, setAcronymList] = useState([]);
+  const [ rowCount, setRowCount ] = useState(5);
```

Comment:
I think we should show more per page
```suggestion
  const [ rowCount, setRowCount ] = useState(10);
```